lib/elastic_whenever/option.rb

```diff
@@ -47,6 +55,18 @@ def initialize(args)
             @variables << { key: key, value: value }
           end
         end
+        opts.on('--assign_public_ip', 'Assign a public IP.') do
+          @assign_public_ip = 'ENABLED'
+        end
+        opts.on('--launch_type launch_type', 'Launch type. Defualt: EC2') do |launch_type|
+          @launch_type = launch_type
+        end
+        opts.on('--security_groups groups', "Example: --security_groups 'sg-2c503655,sg-72f0cb0a'") do |groups|
+          @security_groups = groups
+        end
+        opts.on('--subnets subnets', "Example: --subnets 'subnet-4973d63f,subnet-45827d1d'") do |subnets|
+          @subnets = subnets
+        end
```

Comment:
I guess this option needs validation about launch type. For example, If `security_groups` or `subnets` are unspecified when `launch_type` is FARGATE, will they get an error?

Reply:
I'm still thinking about this/looking into it. fyi, I did discovery something kinda terrible about this. The AWS API does not appear to validate either the subnets or the security groups.. if you specify either a non-existent subnet or security group it will just silently fail.

Reply:
@avinson 	@wata727 	For now it would be best to add a TODO comment to keep it top of mind.